content/en/running/running-on-win.md

```diff
@@ -1,61 +1,72 @@
 ---
-title: Running on Windows
+title: Running on Windows 10/11
 weight: 40
 type: docs
 ---
-## Running on Windows
 
-* Interlisp Online seems to works fine with recent Edge and Chrome; we've tested others but don't regularly use them.
-WARNING: Window/Chrome and Edge use keystrokes (control-W for example) that conflict with Medley use. 
+Medley does not (yet) run as a native Windows application.  However, you can run Medley on Windows 10/11 in one of two ways: via **Windows System for Linux (WSL)** or via the **Docker Desktop for Windows** and the Medley docker image.  Using either of these methods requires the installation (if not already installed) of one additional component to your Windows system - WSL or Docker Desktop.  Once this prerequisite component is installed, there is for each case a single step install for Medley as well as a common `medley` script to run Medley.
 
-* Docker Desktop is a great possibility, too, if your system can run it.
-* Otherwise, to run installed, you either need
-    * WSLg (available with Windows 11) OR
-    * WsL2 A running X-server installed on your machine
+On Windows, Medley requires a 64-bit machine.
```

Comment:
We used to run on 32 bit windows: and with cygwin more recently?

not sure it matters except "what's the cheapest/smallest Lisp machine"
I'd like to get medley 2.0 going for debugging/comparison

Reply:
Right now on Windows our only workable solutions are via WSL and via Docker.  Neither WSL nor Docker are supported (or even run without lots of work) on 32-bit systems.

I wasn't aware that we have a repeatable (i.e., can build with github actions) working cgwin build.  If so, I can weave this into the Windows install/run options.

Longer term, we should just do a native Windows maiko - it couldn't be all that hard.

And, the cheapest Lisp machine is probably a Raspberry PI or one of the million Raspberry Pi clones out there.